tests/Browser/BoardInteractionTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace Tests\Browser;
+
+use App\Models\BoardConfig;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Laravel\Dusk\Browser;
+use Mockery\Exception;
+use Tests\DuskTestCase;
+
+class BoardInteractionTest extends DuskTestCase
+{
+    /**
+     * Test a successful board creation
+     */
+    public function testSuccessfulCreation(): void
+    {
+        $this->browse(function (Browser $browser) {
+            $this->reachDashboard("login");
+
```

Comment:
in the tests below you're testing if the reachDashboard("login") returns a valid user id, here you didn't do that.

Reply:
resolved